### ISSUE TITLE
First time starting way-displays got failed to apply changes

### ISSUE THREAD

**Issue by merc1031** [alex-courtis/way-displays#181]
Now it keeps failing to apply changes as well. Monitors actually look fine, but now there is a large red banner ("Invalid scale passed to monitor eDP-1) following my screens.

I'm using hyprland install from here https://github.com/JaKooLit/Ubuntu-Hyprland/tree/24.04?tab=readme-ov-file on Ubuntu 24.04

The log is here

```
I [10:51:53] way-displays version 1.11.1-SNAPSHOT
I [10:51:53]
I [10:51:53] Found configuration file: /home/lloucas/.config/way-displays/cfg.yaml
I [10:51:53]
I [10:51:53] Active configuration:
I [10:51:53]   Arrange in a ROW aligned at the BOTTOM
I [10:51:53]   Order:
I [10:51:53]     eDP-1
I [10:51:53]     HDMI-A-1
I [10:51:53]     DP-7
I [10:51:53]   Scaling: OFF
I [10:51:53]   Auto scale: OFF (min: 1.000)
I [10:51:53]
I [10:51:53] Monitoring lid device: /dev/input/event1
I [10:51:53]
I [10:51:53] Lid open
I [10:51:53]
I [10:51:53] DP-7 Arrived:
I [10:51:53]   info:
I [10:51:53]     name:      'DP-7'
I [10:51:53]     make:      'LG Electronics'
I [10:51:53]     model:     'LG HDR 4K'
I [10:51:53]     serial:    '203NTBK56789'
I [10:51:53]     desc:      'LG Electronics LG HDR 4K 203NTBK56789 (DP-7)'
I [10:51:53]     width:     700mm
I [10:51:53]     height:    400mm
I [10:51:53]     dpi:       138.25 @ 3840x2160
I [10:51:53]     mode:      3840 x 2160 @  60 Hz   59,997 mHz (preferred)
I [10:51:53]     mode:      3840 x 2160 @  30 Hz   30,000 mHz  29,970 mHz
I [10:51:53]     mode:      2560 x 1440 @  60 Hz   59,951 mHz
I [10:51:53]     mode:      1920 x 1080 @  60 Hz   60,000 mHz  59,940 mHz  59,934 mHz
I [10:51:53]     mode:      1280 x  720 @  60 Hz   60,000 mHz  59,940 mHz
I [10:51:53]     mode:      1024 x  768 @  60 Hz   60,004 mHz
I [10:51:53]     mode:       800 x  600 @  60 Hz   60,317 mHz
I [10:51:53]     mode:       720 x  480 @  60 Hz   60,000 mHz  59,940 mHz
I [10:51:53]     mode:       640 x  480 @  60 Hz   60,000 mHz  59,940 mHz  59,940 mHz
I [10:51:53]   current:
I [10:51:53]     scale:     1.000 (1.440)
I [10:51:53]     size:      3840x2160
I [10:51:53]     position:  7680,0
I [10:51:53]     mode:      3840x2160@60Hz (59,997mHz) (preferred)
I [10:51:53]     VRR:       off
I [10:51:53]
I [10:51:53] HDMI-A-1 Arrived:
I [10:51:53]   info:
I [10:51:53]     name:      'HDMI-A-1'
I [10:51:53]     make:      'LG Electronics'
I [10:51:53]     model:     'LG HDR 4K'
I [10:51:53]     serial:    '201NTYTPD146'
I [10:51:53]     desc:      'LG Electronics LG HDR 4K 201NTYTPD146 (HDMI-A-1)'
I [10:51:53]     width:     700mm
I [10:51:53]     height:    400mm
I [10:51:53]     dpi:       138.25 @ 3840x2160
I [10:51:53]     mode:      3840 x 2160 @  60 Hz   60,000 mHz (preferred)  60,000 mHz  59,940 mHz
I [10:51:53]     mode:      3840 x 2160 @  50 Hz   50,000 mHz
I [10:51:53]     mode:      3840 x 2160 @  30 Hz   30,000 mHz  30,000 mHz  29,970 mHz
I [10:51:53]     mode:      3840 x 2160 @  25 Hz   25,000 mHz
I [10:51:53]     mode:      3840 x 2160 @  24 Hz   24,000 mHz  23,976 mHz
I [10:51:53]     mode:      2560 x 1440 @  60 Hz   59,951 mHz
I [10:51:53]     mode:      1920 x 1080 @  60 Hz   60,000 mHz  59,940 mHz  59,934 mHz
I [10:51:53]     mode:      1920 x 1080 @  50 Hz   50,000 mHz
I [10:51:53]     mode:      1920 x 1080 @  30 Hz   30,000 mHz  29,970 mHz
I [10:51:53]     mode:      1920 x 1080 @  24 Hz   24,000 mHz  23,976 mHz
I [10:51:53]     mode:      1280 x  720 @  60 Hz   60,000 mHz  59,940 mHz
I [10:51:53]     mode:      1024 x  768 @  60 Hz   60,004 mHz
I [10:51:53]     mode:       800 x  600 @  60 Hz   60,317 mHz
I [10:51:53]     mode:       720 x  576 @  50 Hz   50,000 mHz
I [10:51:53]     mode:       720 x  480 @  60 Hz   60,000 mHz  59,940 mHz
I [10:51:53]     mode:       640 x  480 @  60 Hz   60,000 mHz  59,940 mHz  59,940 mHz
I [10:51:53]   current:
I [10:51:53]     scale:     1.500 (1.440)
I [10:51:53]     size:      2560x1440
I [10:51:53]     position:  2560,0
I [10:51:53]     mode:      3840x2160@60Hz (60,000mHz) (preferred)
I [10:51:53]     VRR:       on
I [10:51:53]
I [10:51:53] eDP-1 Arrived:
I [10:51:53]   info:
I [10:51:53]     name:      'eDP-1'
I [10:51:53]     make:      'Samsung Display Corp.'
I [10:51:53]     model:     '0x4165'
I [10:51:53]     desc:      'Samsung Display Corp. 0x4165 (eDP-1)'
I [10:51:53]     width:     340mm
I [10:51:53]     height:    220mm
I [10:51:53]     dpi:       281.98 @ 3840x2400
I [10:51:53]     mode:      3840 x 2400 @  60 Hz   60,000 mHz (preferred)
I [10:51:53]   current:
I [10:51:53]     scale:     2.000 (2.937)
I [10:51:53]     size:      1920x1200
I [10:51:53]     position:  5120,0
I [10:51:53]     mode:      3840x2400@60Hz (60,000mHz) (preferred)
I [10:51:53]     VRR:       off
I [10:51:53]
I [10:51:53] HDMI-A-1 Changing:
I [10:51:53]   from:
I [10:51:53]     scale:     1.500 (1.440)
I [10:51:53]     size:      2560x1440
I [10:51:53]     position:  2560,0
I [10:51:53]     mode:      3840x2160@60Hz (60,000mHz) (preferred)
I [10:51:53]     VRR:       on
I [10:51:53]   to:
I [10:51:53]     VRR:       off
I [10:51:53]
I [10:51:53] HDMI-A-1:
I [10:51:53]   Cannot enable VRR: this display or compositor may not support it.
I [10:51:53]   To speed things up you can disable VRR for this display by adding the following or similar to your cfg.yaml
I [10:51:53]   VRR_OFF:
I [10:51:53]     - 'LG HDR 4K'
I [10:51:53]
I [10:51:53] DP-7 Changing:
I [10:51:53]   from:
I [10:51:53]     scale:     1.000 (1.440)
I [10:51:53]     size:      3840x2160
I [10:51:53]     position:  7680,0
I [10:51:53]     mode:      3840x2160@60Hz (59,997mHz) (preferred)
I [10:51:53]     VRR:       off
I [10:51:53]   to:
I [10:51:53]     position:  7680,240
I [10:51:53]
I [10:51:53] HDMI-A-1 Changing:
I [10:51:53]   from:
I [10:51:53]     scale:     1.500 (1.440)
I [10:51:53]     size:      2560x1440
I [10:51:53]     position:  2560,0
I [10:51:53]     mode:      3840x2160@60Hz (60,000mHz) (preferred)
I [10:51:53]     VRR:       on
I [10:51:53]   to:
I [10:51:53]     scale:     1.000
I [10:51:53]     position:  3840,240
I [10:51:53]
I [10:51:53] eDP-1 Changing:
I [10:51:53]   from:
I [10:51:53]     scale:     2.000 (2.937)
I [10:51:53]     size:      1920x1200
I [10:51:53]     position:  5120,0
I [10:51:53]     mode:      3840x2400@60Hz (60,000mHz) (preferred)
I [10:51:53]     VRR:       off
I [10:51:53]   to:
I [10:51:53]     scale:     1.000
I [10:51:53]     position:  0,0
E [10:51:53]
E [10:51:53] ERROR: Changes failed
E [10:51:53]
E [10:51:53] ERROR: Please raise an issue: https://github.com/alex-courtis/way-displays/issues
E [10:51:53] ERROR: Attach this log and describe the events that occurred before this failure.
```

**Comment by alex-courtis** [comment]
What is the red banner? Is it from Hyprland? Please post a screenshot so that I can attempt to replicate.

It looks like the scale is an issue for the laptop display. You are changing it back to 1, which should not be a problem. Is it actually set to scale 1?
Please attempt two tests to isolate:
1. unplug other monitors
2. remove `Scaling: OFF` and `Auto scale: OFF`

**Comment by alex-courtis** [comment]
How was way-displays installed? I looked at JaKooLit/Ubuntu-Hyprland and JaKooLit/Hyprland-Dots but could not find any references to way-displays.

**Comment by merc1031** [comment]
Sorry lost track of this because it just works now...
I beleive the red banner was from hyprland i was unable to get it to go away without a restart.
I dsiabled Scaling and After a restart it all seemed to work.

> 
How was way-displays installed? I looked at JaKooLit/Ubuntu-Hyprland and JaKooLit/Hyprland-Dots but could not find any references to way-displays.


I installed from source.

I think i'm all good for now. Just the error message asked me to post it here so i figured i'd do so.
